src/engine/renderer/glsl_source/generic_fp.glsl

```diff
@@ -33,8 +33,26 @@ IN(smooth) vec2         var_FadeDepth;
 uniform sampler2D       u_DepthMap;
 #endif
 
+#if !defined(GENERIC_2D) && defined(USE_TCGEN_LIGHTMAP)
+uniform int u_MapOverBrightBitsShift;
```

Comment:
I don't like the name with "shift" as that describes a bit shift e.g. shift by 3 means multiply by 8. How about u_OverbrightMultiplier?

Reply:
I can do that.